src/Datadog.Trace.ClrProfiler.Native/Datadog.Trace.ClrProfiler.Native.DLL.vcxproj

```diff
@@ -113,37 +114,40 @@
     <PreBuildEvent>
       <Message>Remove potentially locked outputs (dll/pdb/xml)</Message>
     </PreBuildEvent>
-    <PostBuildEvent>
-      <Command>IF EXIST $(ProjectDir)\post-build-events-cpp.bat CALL $(ProjectDir)\post-build-events-cpp.bat "$(SolutionDir)"</Command>
-    </PostBuildEvent>
-    <PostBuildEvent>
-      <Message>Copy profiler for other projects</Message>
-    </PostBuildEvent>
+    <PostBuildEvent />
+    <PostBuildEvent />
   </ItemDefinitionGroup>
   <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">
     <ClCompile>
       <WarningLevel>Level3</WarningLevel>
       <Optimization>Disabled</Optimization>
-      <PreprocessorDefinitions>WIN32;_DEBUG;_WINDOWS;_USRDLL;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <SDLCheck>true</SDLCheck>
-      <ConformanceMode>true</ConformanceMode>
       <RuntimeLibrary>MultiThreadedDebug</RuntimeLibrary>
+      <ConformanceMode>true</ConformanceMode>
```

Comment:
Debug|x64 has `<OmitFramePointers>false</OmitFramePointers>` which is missing here. Did you mean to exclude it here?

Reply:
Omitting frame pointers is an optimization that can make debugging harder, so they are omitted (`true`) in Release builds and included (`false`) in Debug builds.
https://docs.microsoft.com/en-us/cpp/build/reference/oy-frame-pointer-omission

Reply:
But `In x64 compilers, /Oy and /Oy- are not available`, which might explain the discrepancy between `x86` and `x64`.